README.md

```diff
@@ -9,41 +9,75 @@ validating international phone numbers. The Java version is optimized for
 running on smartphones, and is used by the Android framework since 4.0 (Ice
 Cream Sandwich).
 
-# Want to report an issue?
-If you want to report an issue, or to contribute to the project, please read
-the guidelines [here](CONTRIBUTING.md) first.
+# Quick links
+
+*   Reporting an issue? Want to send a pull request? See the [contribution
+    guidelines](CONTRIBUTING.md)
+*   Check the [frequently asked questions](FAQ.md)
+*   Fun! [Falsehoods Programmers Believe About Phone Numbers](FALSEHOODS.md)
+*   Look for
+    [`README`s](http://github.com/googlei18n/libphonenumber/find/master) in
+    directories relevant to the code you're interested in.
+*   For contributors and porters: [How to run the Java demo](run-java-demo.md)
+*   For porters: [How to make metadata changes](making-metadata-changes.md)
 
 # Highlights of functionality
-  * Parsing/formatting/validating phone numbers for all countries/regions of the world.
-  * ` getNumberType ` - gets the type of the number based on the number itself; able to distinguish Fixed-line, Mobile, Toll-free, Premium Rate, Shared Cost, VoIP and Personal Numbers  (whenever feasible).
-  * ` isNumberMatch ` - gets a confidence level on whether two numbers could be the same.
-  * ` getExampleNumber `/` getExampleNumberByType ` - provides valid example numbers for all countries/regions, with the option of specifying which type of example phone number is needed.
-  * ` isPossibleNumber ` - quickly guessing whether a number is a possible phonenumber by using only the length information, much faster than a full validation.
-  * ` isValidNumber ` - full validation of a phone number for a region using length and prefix information.
-  * ` AsYouTypeFormatter ` - formats phone numbers on-the-fly when users enter each digit.
-  * ` findNumbers ` - finds numbers in text input.
-  * ` PhoneNumberOfflineGeocoder ` - provides geographical information related to a phone number.
-  * ` PhoneNumberToCarrierMapper ` - provides carrier information related to a phone number.
-  * ` PhoneNumberToTimeZonesMapper ` - provides timezone information related to a phone number.
 
-# Demo (v8.6.0)
-[Java](http://libphonenumber.appspot.com/)
-
-[JavaScript](https://rawgit.com/googlei18n/libphonenumber/master/javascript/i18n/phonenumbers/demo-compiled.html)
-
-# Code
-To include the code in your application, either integrate with Maven or
-download the latest Jars from the Maven repository:
-
-http://repo1.maven.org/maven2/com/googlecode/libphonenumber/libphonenumber/
+*   Parsing / formatting / validating phone numbers for all countries / regions
```

Comment:
Grammarly.com backs me up here :)
"Many people wonder whether to use a space before and after slashes in a sentence. The answer is typically no on both fronts. Spaces before slashes should be avoided. "

Slashes can be followed by space if it is to break up several short-lines in a poem etc.